packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -300,6 +300,16 @@ async function createReferencePriceFeedForEmp(logger, web3, networker, getTime,
     if (!combinedConfig) {
       throw new Error("createReferencePriceFeedForEmp: No default config was found and no user config was provided.");
     }
```

Comment:
This change is made to be compatible with how hardhat treats time within it's test network. Each time a block is mined hard hat's internal clock of time moves forward. This cant be disabled. This breaks a bunch of bot logic and price feeds as they rely on the internal time of the bot and relies on that being close to the passage of block time. To provide compatibility, this override is added to enable use the block time as the bot's clock.